bin/cluster

```diff
@@ -290,6 +291,8 @@ if __name__ == '__main__':
                                help='number of infra nodes to create in cluster')
     create_parser.add_argument('-e', '--etcd', default=0, type=int,
                                help='number of external etcd hosts to create in cluster')
+    create_parser.add_argument('--env', default='prod', type=str,
```

Comment:
@kwoodson agreed, but bin/cluster is used beyond our internal usage. Prod just seems like an odd default for tooling meant to operate on more than one 'environment'